e2e/testdata/pod-pvc-template.yaml

```diff
@@ -0,0 +1,47 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ .pvcName }}
+  namespace: {{ .namespace }}
+{{ if eq .useAnnotation "true" }}
+  annotations:
+    resize.topolvm.io/threshold: {{ if ne .thresholdAnnotation "" }}{{ .thresholdAnnotation }}{{ else }}""{{ end }}
```

Comment:
Can this be `"{{.thresholdAnnotation}}"`?

Reply:
Thank you. I fixed it.